src/chart.js

```diff
@@ -19,6 +19,9 @@ require('./core/core.legend')(Chart);
 require('./core/core.interaction')(Chart);
 require('./core/core.tooltip')(Chart);
 
+// By default, we only load the browser platform.
+Chart.platform = require('./platforms/platform.dom')(Chart);
```

Comment:
Should it be `Chart.Platform` or `Chart.platform`? I thought namespaces was all lowercase and classes with a capital.